Tests/SwiftGenKitTests/ColorsCLRFileTests.swift

```diff
@@ -10,30 +10,30 @@ import SwiftGenKit
 
 class ColorsCLRFileTests: XCTestCase {
   func testEmpty() {
-    let parser = ColorsCLRFileParser()
+    let parser = ColorsFileParser()
 
     let result = parser.stencilContext()
     XCTDiffContexts(result, expected: "empty.plist", sub: .colors)
   }
 
   func testFileWithDefaults() throws {
-    let parser = ColorsCLRFileParser()
+    let parser = ColorsFileParser()
```

Comment:
Hmm, right. But then how do you test the parser routing, namely that it invokes the right bits?
(I have the same question for the SwiftGen CLI, where we currently don't have any tests...)